### ISSUE TITLE
[Bug]:Files list scrolls back to a first opened file. (Intoduced only in ver 28)

### ISSUE THREAD

**Issue by tambel** [nextcloud/server#44300]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on [Github](https://github.com/nextcloud/server/issues?q=is%3Aopen+is%3Aissue+label%3Abug) OR [Nextcloud Community Forum](https://help.nextcloud.com/) _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

_To be honest, there's a semi-related [topic on the community forum](https://help.nextcloud.com/t/viewing-photos-via-nextcloud-files-if-i-scroll-down-and-open-and-close-a-photo-it-always-brings-me-back-to-the-top/179960?u=tambel) but i doubt that it got any attention, so I think it worth risking and also making it here in order to smash that nasty bug, which, somehow slipped into a new version._

Description:
If you go to some folder and open some file (lets say FILE1), then open another file, after closing it, the file list, for some reason, changes it's focus to the FILE1 again.
 

### Steps to reproduce

1. Go to a folder with enough files to be able to scroll though it.
2. Open some file at the beginning of a list. 
3. Close it.
4. Open another file which is at the end of a list, then also close it.
5. **List's focus is switched back to the beginning to a first file.**

[ezgif-3-8b236480c5-ezgif.com-gif-to-webm-converter.webm](https://github.com/nextcloud/server/assets/10327167/c1684e04-8b61-4071-8015-11b39d620c3c)

In the GIF, you can see that the focus of the files list keeps returning to the `Birdie TEST FILE1.jpg` file, even if another file was opened.

Moreover, there are also some other effects that can be seen on the GIF after that.

1. If you refresh the files list page, new page will automatically open `Birdie TEST FILE1.jpg`.

2. If you navigate back from the folder. You will get the File not found error message.

### Expected behavior

After closing a second file, lists focus shouldn't change, and refreshing the page shouldn't reopen first file.

### Installation method

Other Community project

### Nextcloud Server version

28

### Operating system

None

### PHP engine version

None

### Web server

None

### Database engine version

None

### Is this bug present after an update or on a fresh install?

None

### Are you using the Nextcloud Server Encryption module?

None

### What user-backends are you using?

- [ ] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Additional info

Installed as a Truenas SCALE app. 
App version: 1.6.58 (Nextcloud 28.0.3 )

**Comment by skjnldsv** [comment]
> _To be honest, there's a semi-related [topic on the community forum](https://help.nextcloud.com/t/viewing-photos-via-nextcloud-files-if-i-scroll-down-and-open-and-close-a-photo-it-always-brings-me-back-to-the-top/179960?u=tambel) but i doubt that it got any attention, so I think it worth risking and also making it here in order to smash that nasty bug, which, somehow slipped into a new version._

You did right, thanks for reporting :)

**Comment by qibao07** [comment]
NC29 has the same problem.

These may be the same problem. #46084 #44792

**Comment by qibao07** [comment]
I found a temporary solution.

```
sudo sed -i 's/{this.scrollToFile(t,!1)}/{\/\*this.scrollToFile(t,!1)\*\/}/g' {PATH}/nextcloud/dist/files-main.js
```

**Comment by joshtrichards** [comment]
Fixed in #48632
